### DISCUSSION TITLE
Payment methods for Mollie gateway

### DISCUSSION THREAD

**Discussion by justkidding96** [duncanmcclean/simple-commerce#881] [Feature Requests]
Hi I'm missing the option to render the available payment methods from Mollie. Also the current Mollie gateways redirects you to the Mollie website. Is it possible to extend the default gateway with more options like above?

**Comment by duncanmcclean** [comment]
Hello,

You should be presented with the additional payment options that Mollie offers after you've been redirected to Mollie. They won't show up in the dropdown of gateways on the payment checkout page.

You can change the name of the Mollie gateway to be more user-friendly if you wish:

```php
// config/simple-commerce.php

'gateways' => [
    \DoubleThreeDigital\SimpleCommerce\Gateways\Builtin\MollieGateway::class => [
        // Other settings...
        'display' => 'Mollie (Card / Klarna / PayPal / etc)',
    ],
],
```

**Comment by justkidding96** [comment]
@duncanmcclean Yes I understand. But in Woocommerce you can display the icons which Mollie provide. Would it be possible to implement that you can access the payment methods?

**Reply by duncanmcclean** [reply]
As I believe it, Mollie allows you to enable/disable payment methods on an account-basis. So the payment methods enabled for one site, might not be for another so it's likely not something I'll add into the starter kit.

However, the checkout templates are inside your own `resources/views` directory so you can customise them however you'd like (by adding logos or anything else).
